public/locales/en-US/translations.json

```diff
@@ -19,7 +19,7 @@
   "explorer": "Explorer",
   "xrpl_org": "XRPL.org",
   "github": "GitHub",
-  "header.search.placeholder": "Search by Address, Ledger or Txn",
+  "header.search.placeholder": "Search by Token, Address, Ledger or Txn",
```

Comment:
I think the translation for this should be set to `null` in other translations so maintainers would go in later and make changes. Also, users in different languages would get announced about the new feature even though it would be in English. @ckniffen let us know if you think otherwise

Reply:
You might message a few people like those that worked on the translations to find the correct wording since this is text that is usually visible on the screen.